README.md

```diff
@@ -426,3 +427,7 @@ We welcome pull requests!
 - Your PR is more likely to be accepted if it includes tests. (We have not historically been very strict about tests, but we would like to improve this!).
 - You're welcome to submit a draft PR if you would like early feedback on an idea or an approach.
 - Happy coding!
+
+## License Agreement
+
+This entire project comes with a Apache License, Version 2.0. Kindly read it [here](LICENSE)
```

Comment:
Please remove this, we don't need to duplicate the information that's already covered by License